dom/nodes/Node-appendChild-script-and-button-from-div.html

```diff
@@ -0,0 +1,33 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>Node.appendChild: inserting script and button from a div</title>
+<link rel=help href="https://dom.spec.whatwg.org/#dom-node-appendChild">
+<link rel=help href="https://dom.spec.whatwg.org/#concept-node-insert">
+<link rel=help href="https://github.com/whatwg/dom/issues/575">
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<div id="log"></div>
+</head>
+<body>
+<form id="form"></form>
+<script>
+var form = document.getElementById("form");
+
+var script = document.createElement("script");
+var scriptRan = false;
+script.textContent = `
+    assert_equals(button.form, null, "button already has a form owner");
```

Comment:
So this is basically assuming the maybe-WebKit model whereby the script is executed as part of the insertion steps rather than after as part of its own loop?

Reply:
Yes, script preparation can happen when an element [becomes connected](https://html.spec.whatwg.org/multipage/scripting.html#script-processing-model:becomes-connected).